README.md

```diff
@@ -18,6 +18,48 @@ npm i @fullstory/browser --save
 yarn add @fullstory/browser
 ```
 
+## Migrating to Version 2.0.0
+In version 2.0.0, `init` is a separate named export from `FullStory`. You will need to update all of your wildcard (`'*'`) imports to explicit named imports.
+
+_Version 1.x.x_
+```js
+import * as FullStory from '@fullstory/browser';
+```
```

Comment:
Unclear where the "Migrating to Version 2.0.0"  section should be. ALSO: should this say `2.x.x`? `^2.0.0`? (I overthink things 😂 )

Reply:
I think fine to leave where you have it. I also think `Version 2.x.x` is good